### DISCUSSION TITLE
Connector Registration Error

### DISCUSSION THREAD

**Discussion by sachinkumarUI** [International-Data-Spaces-Association/DataspaceConnector#103] [Q&A] [answered]
Hi,

While registering my connector  at https://ids.broker.mobilitydataspace.io/infrastructure I am getting the below error. The registration used to work fine in the past, I started experiencing the below error from 08/10/2021

{"reason":{"properties":null,"@id":"https://w3id.org/idsa/code/INTERNAL_RECIPIENT_ERROR"},"payload":"Failed to retrieve own DAPS token, preventing a valid response.","type":"de.fraunhofer.iais.eis.RejectionMessageImpl"}

Kindly advise,

Thanks in advance

**Comment by juliapampus** [comment, marked as answer]
Hi @sachinkumarUI, does this error still exist? Is your IDS certificate still valid?
